lib/slack.rb

```diff
@@ -1,11 +1,158 @@
 #!/usr/bin/env ruby
+require "pry"
+
+require_relative "channel"
+require_relative "user"
+require "httparty"
+require "dotenv"
+Dotenv.load
+
+class SlackError < StandardError; end
+
+class Slack
+  attr_reader :channels, :users
+
+  def initialize()
+    @channels = []
+    @users = []
+    get_lists
+  end
+
+  def get_lists()
```

Comment:
Just notice here how the loops look almost identical.  This is a good indication that you can dry things up using duck typing / polymorphism.